Sources/Target/Linux/Process.cpp

```diff
@@ -363,6 +364,7 @@ ErrorCode Process::getMemoryRegionInfo(Address const &address,
       //
       info.start = last;
       info.length = start - last;
+      info.name = std::string(name);
```

Comment:
You don't need to explicitly call the `std::string` constructor here I think. You can just do `info.name = name;`.